csgc-jekyll-src/resources.md

```diff
@@ -6,28 +6,21 @@ permalink: /resources/
 
 ## Points of contact 
 
-* Please don't hesitate to [email any of us](/Officers/)!
```

Comment:
Maybe we should keep this link, since it's a dedicated "points of contact" section.

Reply:
Either that, or call it a "Social Media" section and include a link to the Slack channels that Steve started.

Reply:
On further thought, I suggest we make this a "Social Media" section and include a link to the Slack page.

Reply:
Good suggestions! Myles, let's go ahead and do that.